reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -243,72 +206,59 @@ for file objects.
 </Types>
 ```
 
-After you save the file and close it, run an `Update-TypeData` command,
-such as the following command, to add the new Types.ps1xml file to the
-current session. The command uses the `PrependData` parameter to place the
-new file in a higher precedence order than the original file. (For more
-information about `Update-TypeData`, see
-[Update-TypeData](../../Microsoft.PowerShell.Utility/Update-TypeData.md).)
+Run `Update-TypeData` to add the new `Types.ps1xml` file to the current
+session. The command uses the **PrependData** parameter to place the new file
+in a higher precedence order than the original file.
+
+For more information about `Update-TypeData`, see
+[Update-TypeData](../../Microsoft.PowerShell.Utility/Update-TypeData.md).
 
 ```powershell
 Update-Typedata -PrependPath $PSHOME\MyTypes.ps1xml
 ```
 
-To test the change, run a `Get-ChildItem` command to get the
-PowerShell.exe file in the `$PSHOME` directory, and then pipe the file to
-the `Format-List` cmdlet to list all of the properties of the file. As a
-result of the change, the `Age` property appears in the list.
+To test the change, run a `Get-ChildItem` command to get the PowerShell.exe
+file in the `$PSHOME` directory, and then pipe the file to the `Format-List`
+cmdlet to list all of the properties of the file. As a result of the change,
+the **Age** property appears in the list.
 
 ```powershell
-Get-ChildItem $PSHOME\PowerShell.exe | Format-List -Property *
-
-PSPath            : Microsoft.PowerShell.Core\FileSystem::C:\WINDOWS...
-PSParentPath      : Microsoft.PowerShell.Core\FileSystem::C:\WINDOWS...
-PSChildName       : PowerShell.exe
-PSDrive           : C
-PSProvider        : Microsoft.PowerShell.Core\FileSystem
-PSIsContainer     : False
-Age               : 16
-VersionInfo       : File:             C:\WINDOWS\system32\WindowsPow...
-InternalName:     POWERSHELL
-OriginalFilename: PowerShell.EXE
-# ...
+Get-ChildItem $PSHOME\powershell.exe | Select-Object Age
 ```
 
-You can also display the `Age` property of the file by using the following
-command.
-
-```powershell
-(Get-ChildItem $PSHOME\PowerShell.exe).Age
-# 16
+```Output
+142
 ```
 
 ## The XML in Types.ps1xml Files
```

Comment:
Should the file have ticks around it (this was changed in a different H2 in this PR)?

Reply:
Fixed